packages/dappmanager/src/modules/stakerConfig/params.ts

```diff
@@ -0,0 +1,36 @@
+import { Network } from "@dappnode/common";
+import {
+  ExecutionClientOrSignerVersion,
+  LodestarStakersMinimumVersions
+} from "./types";
+
+const lodestarStakersPraterMinimumVersions: ExecutionClientOrSignerVersion<"prater"> =
+  {
+    "goerli-nethermind.dnp.dappnode.eth": "1.0.8",
+    "goerli-geth.dnp.dappnode.eth": "0.4.28",
+    "goerli-erigon.dnp.dappnode.eth": "0.1.5",
+    "goerli-besu.dnp.dappnode.eth": "0.1.1",
+    "web3signer-prater.dnp.dappnode.eth": "0.1.18"
+  };
+
+const lodestarStakersGnosisMinimumVersions: ExecutionClientOrSignerVersion<"gnosis"> =
+  {
+    "nethermind-xdai.dnp.dappnode.eth": "1.0.29", // Needs publishment
+    "web3signer-gnosis.dnp.dappnode.eth": "0.1.13" // Needs publishment
```

Comment:
0.1.13 is already published but does not yet include the Lodestar api token

```suggestion
    "web3signer-gnosis.dnp.dappnode.eth": "0.1.14" // Needs publishment
```